digitalocean/database/resource_database_redis_config_test.go

```diff
@@ -9,6 +9,7 @@ import (
 )
 
 func TestAccDigitalOceanDatabaseRedisConfig_Basic(t *testing.T) {
+	t.Skip("Skipping redis cluster tests.TODO:- remove them")
```

Comment:
Just wanted to note that there are additional Redis tests in the tests for `resource_database_cluster`.

https://github.com/digitalocean/terraform-provider-digitalocean/blob/d3593580cdf22c054abbc1c92f53befce70b7ec7/digitalocean/database/resource_database_cluster_test.go#L286